cve-whitelist.yaml

```diff
@@ -0,0 +1,3 @@
+generalwhitelist:
+  CVE-2019-5482: curl
```

Comment:
You've replaced curl with wget in the script -- do we still need this then?

Reply:
yes, we need `CVE-2019-5482: curl` to whitelist `git` dependency `libcurl3-gnutls`

Reply:
If we're ignoring this then then we may as well stick with curl to install instead of wget?

Reply:
yes

Reply:
fixed.